sdk/core/azure-core/src/datetime.cpp

```diff
@@ -50,6 +52,9 @@ struct ComputeYearResult
   int SecondsLeftThisYear;
 };
 
+constexpr auto MinYear = 1601;
+constexpr auto MaxYear = 9999;
```

Comment:
I don't think these need to be public. we can just document the range.

Reply:
I agree. However, they aren't public. Just named constants as part of implementation detail.